superset/assets/javascripts/modules/superset.js

```diff
@@ -141,10 +141,11 @@ const px = function () {
         let msg = '';
         if (!xhr.responseText) {
           const status = xhr.status;
-          msg += 'An unknown error occurred. (Status: ' + status + ')';
           if (status === 0) {
             // This may happen when the worker in gunicorn times out
-            msg += ' Maybe the request timed out?';
+            msg += 'Could not reach server';
```

Comment:
I don't want it to be too long though as it may repeat on all slices in the case of the dashboard view. It doesn't look nice to have a long error message shown in a bunch of places in the same view...